core/Core.lua

```diff
@@ -117,6 +117,14 @@ function addon:OnInitialize()
 	end
 	--@end-alpha@
 
+	-- Disable various tutorial windows
+	C_CVar.SetCVar("showTutorials", 0)
+	C_CVar.SetCVar("showNPETutorials", 0)
```

Comment:
We probably shouldn't disable these as they're pretty broad in scope.

On a side note, I didn't know what NPE was referring to here and it turns out, it's the New Player Experience that was added in Shadowlands.

Reply:
I went back and forth on this one a lot, and I really agree with you here -- I was worried about the scope of these being way, way outside what a bag addon should do.

I think I'm going to pull back this change tomorrow -- I can't reproduce the reagent bag message anymore for whatever reason. Because we do a RawHook on the open bag function calls, the code to clear the tutorial message never gets called. I tried digging around, but I couldn't find the specific cvar that would disable this message entirely.

Reply:
If I'm not mistaken, [this](https://github.com/Gethe/wow-ui-source/blob/532edb9c5c2930dcc60a2f59c48f8fb414652be1/Interface/AddOns/Blizzard_Tutorials/Blizzard_TutorialReagentBag.lua#L56) is the offending flag for clearing the reagent bag's tutorial.

Additionally, [this function](https://github.com/Gethe/wow-ui-source/blob/bdf9c61809fe5d30bf266e8961e1e8cb8f52499e/Interface/AddOns/Blizzard_Tutorials/Blizzard_Tutorials_Professions.lua#L246-L266) appears to be responsible for the behavior reported in #800—with [this one](https://github.com/Gethe/wow-ui-source/blob/bdf9c61809fe5d30bf266e8961e1e8cb8f52499e/Interface/AddOns/Blizzard_Tutorials/Blizzard_Tutorials_Professions.lua#L214-L218) neutralizing it.

Reply:
Oh interesting, this looks correct. I was looking at https://wowpedia.fandom.com/wiki/CVar_closedInfoFrames, which seems to be out of date.

I'll look at the #800 issue in another commit.

Thank you for the tip here!